osu.Game/Screens/SelectV2/PanelBase.cs

```diff
@@ -61,6 +60,12 @@ public Color4? AccentColour
             }
         }
 
+        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) =>
+            TopLevelContent.ReceivePositionalInputAt(screenSpacePos);
```

Comment:
What does this do?

Reply:
There's an X offset applied to this child, without this, input gets handled to the left of panels even though there's nothing but empty space.

Reply:
Would probably appreciate an inline comment about this at least.

Reply:
Added in force push.